templates/Dataland/v2/download-card.html.twig

```diff
@@ -33,57 +32,58 @@
                     Files
                 </div>
                 <div class="card-body">
-                        {% if dataset.datasetSubmission.nceiurl|default %}
-                            <div>
-                                <a href="{{ dataset.datasetSubmission.nceiurl }}" target="_blank">
-                                <button type="button" class="btn btn-primary btn-sm">
-                                    <i class="fas fa-external-link-alt"></i>
-                                NCEI
-                                </button>
-                                </a>
-                            </div>
-                            <hr>
-                        {% endif %}
-                        {% if dataset.datasetSubmission.erddapurl|default %}
-                            <div>
-                                <a href="{{ dataset.datasetSubmission.erddapurl }}" target="_blank">
-                                <button type="button" class="btn btn-primary btn-sm">
-                                    <i class="fas fa-external-link-alt"></i>
-                                ERDDAP
-                                </button>
-                                </a>
-                            </div>
-                            <hr>
-                        {% endif %}
-                    <div id="download-zip" data-id="{{ dataset.id }}"></div>
-                    <hr>
+                    {% if dataset.availabilityStatus == constant('App\\Entity\\DatasetSubmission::AVAILABILITY_STATUS_PUBLICLY_AVAILABLE') %}
+                        <div id="download-zip" data-id="{{ dataset.id }}"></div>
```

Comment:
I think need to leave this out, or else the component might have issues. Anyways, it has it own logic.